var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -23,14 +23,23 @@ class Pango(AutotoolsPackage):
 
     variant('X', default=False, description="Enable an X toolkit")
 
-    depends_on("pkgconfig", type="build")
+    depends_on("pkgconfig@0.9.0:", type="build")
     depends_on("harfbuzz")
     depends_on("cairo")
     depends_on("cairo~X", when='~X')
     depends_on("cairo+X", when='+X')
     depends_on("libxft", when='+X')
-    depends_on("glib")
+    depends_on('glib')
     depends_on('gobject-introspection')
+    depends_on('fontconfig')
+    depends_on('freetype@2:')
+    depends_on('libffi')
+
+    depends_on('pkgconfig@0.9.0:', type="build", when='@1.41.0')
```

Comment:
@scheibelp Good catch! I'll remove the version specifier from the first `depends_on("pkgconfig@0.9.0:", type="build")` directive.

Reply:
Rebased on develop and removed version specifier as noted in previous comment in https://github.com/spack/spack/pull/12384/commits/9bc3a8b98d469399ad5c368ccc6caccfc1daff6e.

Reply:
@goxberry apologies but I'm confused: when you said

> I'll remove the version specifier from the first `depends_on("pkgconfig@0.9.0:", type="build")` directive

I would have assumed that it would either end up as 

```
depends_on('pkgconfig', type="build", when='@1.41.0')
```

Or as 

```
depends_on('pkgconfig@0.9.0:', type="build")
```

But currently both of the parts I would consider version specifiers still remain. That is fine to me (and from my perspective this looks all good) but I couldn't sync it with https://github.com/spack/spack/pull/12384#discussion_r319407135 so I wanted to make sure this was the intent.

Reply:
@scheibelp The idea was to assume that many versions of the package require `pkgconfig`, hence `depends_on('pkgconfig', type='build')`, and then add additional constraints on `pkgconfig` versions for `pango@1.41.0` via `depends_on('pkgconfig@0.9.0:', type='build', when=@1.41.0)` because I know what the lower bound on the `pkgconfig` version should be for `pango@1.41.0`, whereas I do not know what the lower bound on the `pkgconfig` version should be for other versions of `pango`.

Reply:
Let's just assume the lower bound for pkgconfig is the same for all versions. People rarely change the pkgconfig version requirement in their packages.

Reply:
@adamjstewart @scheibelp Done in https://github.com/spack/spack/pull/12384/commits/975512e1b27f08cf9d7b21c5ca2ad3aad6c07377.